scripts-available/CDB_Groups.sql

```diff
@@ -50,37 +50,45 @@ BEGIN
 END
 $$ LANGUAGE PLPGSQL VOLATILE;
 
--- Adds a user to a group
+-- Adds users to a group, comma-separated
 CREATE OR REPLACE
-FUNCTION cartodb.CDB_Group_AddMember(group_name text, username text)
+FUNCTION cartodb.CDB_Group_AddUsers(group_name text, usernames text)
```

Comment:
you might want to change the signature to `cartodb.CDB_Group_AddUsers(group_name text, usernames text[])` so that you don't need to `string_to_array`.